source/wp-content/themes/wporg-parent-2021/patterns/post-navigation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Title: Post Navigation
+ * Slug: wporg-parent-2021/post-navigation
+ * Inserter: no
```

Comment:
```suggestion
 * Category: wporg
```

Since this isn't a content-syncing pattern, it could be used in the editor (even just by Neso building pages). Removing `Inserter: no` doesn't stop you from still using it in `<!-- wp:pattern` code.

Reply:
Got it. Makes sense.